tsung/erlang_subst/generator.erl

```diff
@@ -0,0 +1,96 @@
+-module(generator).
+-export([uuid/1]).
+-export([case_json/1]).
+
+uuid({Pid, DynData}) ->
```

Comment:
huge hack, but i figure it's fine for this. config for loading external modules seemed like a lot